stack/base-with-services/Dockerfile

```diff
@@ -58,4 +65,4 @@ USER ${NB_USER}
 WORKDIR "/home/${NB_USER}"
 
 # Initialize the database
-RUN mamba run -n aiida-core-services initdb -D aiida_db -U aiida
+RUN mamba run -n aiida-core-services initdb
```

Comment:
What is the DB name used when create aiida profile after this change? We set it explicitly in `config-quick-setup.yaml` as `aiida_db`. 

BTW, we need keep in mind to adapt the change to use `verdi presto` after we move to 2.6.0

Reply:
Good question. `-D` parameter doesn't set the db name, it sets the DB location.

https://www.postgresql.org/docs/current/app-initdb.html#APP-INITDB-OPTION-PGDATA

 in this case it would set it to `~/aiida_db`. But this was broken anyway, since the location in `stack/base-with-services/before-notebook.d/20_start-postgresql.sh` was set to `~/.postgresql`, so essentially this initdb command was being ignored. Now the location is set via the PGDATA variable and thus is guaranteed to be the same everywhere to prevent these kind of mismatches in the future.

Note that the the `-U` parameter sets the so-called [bootstrap superuser](https://www.postgresql.org/docs/current/glossary.html#GLOSSARY-BOOTSTRAP-SUPERUSER) I am not sure if this would have any effect, but we are not using this parameter in `stack/base-with-services/before-notebook.d/20_start-postgresql.sh` so I deleted it here as well.

Reply:
> BTW, we need keep in mind to adapt the change to use verdi presto after we move to 2.6.0

I don't think we'll want to use `verdi presto` since we'd loose some config options. Instead I believe we should migrate from `verdi quicksetup` to the full `verdi profile setup`. But that's not that urgent, quicksetup has been deprecated but not removed.

Reply:
Makes sense, thanks!